.github/workflows/bump-controls.yml

```diff
@@ -0,0 +1,112 @@
+# Copyright (c) Files Community
+# Licensed under the MIT License.
+
+# Abstract:
+#  Bumps the version of the Files UI Controls library
+#
+# Workflow:
+#  1. Find the version of the Files UI Controls library
+#  2. Bump the version of the Files UI Controls library
+#  3. Commit the changes to a new branch in the repo
+#  4. Push the changes to the repo
+#  5. Create a pull request and request a review
+
+name: Bump Files UI Controls
+
+on:
+  workflow_dispatch:
+  pull_request:
+    branches: [ "main" ]
+
+jobs:
+  bump:
+    runs-on: windows-latest
+    strategy:
+      fail-fast: false
+    env:
+      WORKING_DIR:                '${{ github.workspace }}' # D:\a\Files\Files\
+      PROPS_PATH:                 '${{ github.workspace }}\src\Files.App.Controls\CurrentVersion.props'
+
+    steps:
+      - name: Checkout the repository
+        uses: actions/checkout@v4
+
+      - name: Generate GitHub Apps token
+        if: github.repository_owner == 'Lamparter'
```

Comment:
This should probably be switched to files-community.

Reply:
I'm aware, this is one of the things that needs to be done right before merge. Changing this would mean I can't test it.

Reply:
If everything else is ready, we should make this change and merge

Reply:
Sure
Fingers crossed it works